routers/movie.router.js

```diff
@@ -5,8 +5,20 @@ const auth = require('../config/auth').auth;
 var reviewRouter = require('./review.router');
 
 router.get('/', async (req, res) => {
-    const movies = await movieController.getAll();
-    res.json(movies);
+    // const movies = await movieController.getAll();
+    // res.json(movies);
+    const pagination = {
+        sortBy: _.get(req.query, 'sortBy', 'createdAt'), // createdAt, rating
+        direction: _.get(req.query, 'direction', 'desc'), // desc, asc
+        limit: parseInt(_.get(req.query, 'limit', '3')), // any number > 0
+        page: parseInt(_.get(req.query, 'page', '1')), // any number >= 1
+    };
+    const filter = {
+        averageRating: parseInt(_.get(req.query, 'filterRating', '0')),
+        genre: _.get(req.query, 'filterGenre', null)
```

Comment:
We expect a value like `horror,comedy` from the request query. Hence, we'll need to convert the string value into an array like so:
`genres: _.get(req.query, 'filterGenres', '').split(',')`